### ISSUE TITLE
"Enchancement" : cellEdited only to specific cells

### ISSUE THREAD

**Issue by Rodbjartson** [olifolkerd/tabulator#847]
Hey!

I suggest we add functionality to only trigger the celledited for the cells we want to have this behavior.

Say you have a table with 10 columns and you only want a celledited callback for ONE of these columns. Its quite inefficient to trigger this callback and run the code if you only want one of these columns to trigger.

Specific example:
Table with 10 columns, and one is called totalcost.
When you change a cell within the totalcost column, you want the program to update the database straight away with a new totalcost value.

Would it not be nice if we could provide parameters/options to restrict the code to be run?

Like:

```
cellEdited: function (cell) { ... } //triggers whenever a cell is edited
cellEdited: function (cell, totalCost) { ... } //triggers only when a cell within the totalcost column gets edited
```

**Comment by olifolkerd** [comment]
Hey @Rodbjartson 

Thanks for the suggestion, though to fit with convention, in this case it would need to be declared in the column definition object.

In the 3.4 branch i have just pushed an update to allow a ***cellEdited*** callback to be used in a column definition:

```js
{title:"Name", field:"name", cellEdited:function(cell){
        //cell - cell component
    },
}
```

This will be included in next months 3.4 release.

Thanks for the great suggestion, keep the ideas coming.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Tabulator version 3.4 has now been officially release!

You can find details of the new release in the [News Section](http://tabulator.info/news)

Full documentation of all the new features can be found [Here](http://tabulator.info/docs/3.4)

Cheers

Oli :)
